.ado/templates/android-dep-setup.yml

```diff
@@ -0,0 +1,40 @@
+parameters:
+  # latest gradlew use this NDK version, which is not avaialble with any recent agents in azure
+- name: ndkVersion
+  type: string
+  default: '21.0.6113669'
+steps:
+  - task: Bash@3
+    displayName: 'NDK Download'
+    inputs:
+      targetType: 'inline'
+      # install a compatible version of NDK for gradle
+      script: echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install 'ndk;${{ parameters.ndkVersion }}'
+  
+  - task: Bash@3
+    displayName: 'Android Emulator Setup'
+    inputs:
+      targetType: 'inline'
+      script: |
+        #!/usr/bin/env bash
+        
+        # Install AVD files
+        echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install 'system-images;android-27;google_apis;x86'
+        
+        # Create emulator
+        echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n android_emulator -k 'system-images;android-27;google_apis;x86' --force
```

Comment:
Mostly out of curiosity, whats the minimum API we (Microsoft / Office) plans to support?
For macOS it's N-2 from the latest (so macOS 10.14, 10.15, and 11)
for iOS, it's N-1 (iOS 13 and 14)

Reply:
right now we are aiming to support min(office/outlook) which is 23
it is a bit complicated in android , and there is no thumb rule across.